openapi/components/schemas/Organizations/OrganizationSettings.yaml

```
@@ -25,3 +25,21 @@ properties:
       Available only for US-based organizations.
     items:
       $ref: ./TaxLocation.yaml
+  notifications:
+    type: object
+    description: The settings for notifications.
```

nitpick: I think it might be good to be explicit here

```suggestion
    description: Notification settings for the organization.
```